tests/components/bond/test_fan.py

```diff
@@ -90,3 +96,46 @@ async def test_update_reports_fan_off(hass: core.HomeAssistant):
         await hass.async_block_till_done()
 
     assert hass.states.get("fan.name_1").state == "off"
+
+
+async def test_update_reports_direction_forward(hass: core.HomeAssistant):
+    """Tests that update command sets correct direction when Bond API reports fan direction is forward."""
+    await setup_platform(hass, FAN_DOMAIN, ceiling_fan("name-1"))
+
+    with patch(
+        "homeassistant.components.bond.Bond.getDeviceState",
+        return_value={"direction": Directions.FORWARD},
+    ):
+        async_fire_time_changed(hass, utcnow() + timedelta(seconds=30))
+        await hass.async_block_till_done()
+
+    assert hass.states.get("fan.name_1").attributes[ATTR_DIRECTION] == DIRECTION_FORWARD
+
+
+async def test_update_reports_direction_reverse(hass: core.HomeAssistant):
+    """Tests that update command sets correct direction when Bond API reports fan direction is reverse."""
+    await setup_platform(hass, FAN_DOMAIN, ceiling_fan("name-1"))
+
+    with patch(
+        "homeassistant.components.bond.Bond.getDeviceState",
+        return_value={"direction": Directions.REVERSE},
+    ):
+        async_fire_time_changed(hass, utcnow() + timedelta(seconds=30))
+        await hass.async_block_till_done()
+
+    assert hass.states.get("fan.name_1").attributes[ATTR_DIRECTION] == DIRECTION_REVERSE
+
+
+async def test_set_fan_direction(hass: core.HomeAssistant):
+    """Tests that turn off command delegates to API."""
```

Comment:
stale comment here :)

Reply:
thanks! fixed

Reply:
> stale comment here :)

CI build failed for unrelated reason. @ctalkington do you know if there is away to re-trigger it? Or should I just push a noop commit to do it?